### PR TITLE
Feat/cs 43917 equalTo query operator

### PR DESCRIPTION
**Pull request by harshithad0703**
implemented equalTo and referenceIn query operator implementation

### CODE REVIEW

File: test/api/entry-queryables.spec.ts [hunk 1]
```diff
@@ -94,6 +94,28 @@ describe('Query Operators API test cases', () => {
...
+    
+      if (query.entries) {
+        expect(query.entries[0]._version).toBeDefined();
+        expect(query.entries[0].locale).toBeDefined();
+        expect(query.entries[0].uid).toBeDefined();
+        expect(query.entries[0].title).toBe('value');
+      }
+    });
+
+    test('should return entry for referencedIn query', async () => {
```
Comment (nadeem-cs) [inline test/api/entry-queryables.spec.ts:109]: @harshithad0703 Change test to it method to maintain consistency

File: test/unit/entry-queryable.spec.ts [hunk 2]
```diff
@@ -43,4 +43,15 @@ describe('Query Operators API test cases', () => {
       const query = await contentType.Entry().query().and(query1, query2);
       expect(query._parameters).toStrictEqual({ '$and': [ {'fieldUID': {'$in': ['value']}}, { 'fieldUID': 'value2' } ] });
     });
+    it('should return entry equal to the condition - equalTo', async () => {
+      const query = contentType.Entry().query().equalTo('fieldUID', 'value');
+      expect(query._parameters).toStrictEqual({ 'fieldUID': 'value' });
+    });
+    test('should return entry for referencedIn query', async () => {
```
Comment (nadeem-cs) [inline test/unit/entry-queryable.spec.ts:50]: @harshithad0703 change this one too

**Review by nadeem-cs — CHANGES_REQUESTED** [review]

**Review by nadeem-cs — APPROVED** [review]
Approved.

File: test/unit/entry-queryable.spec.ts (hunk 2 above)
Comment (harshithad0703) [reply test/unit/entry-queryable.spec.ts]: changed

**Pull request merged**